src/TicTacToe.Console/Extensions/CollectionExtension.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace TicTacToe.Console.Extensions
+{
+    public static class CollectionExtension
+    {
+        public static void ForEach<TElement>(this ICollection<TElement> collection, Action<TElement> action)
```

Comment:
1. Please, move all extensions to separate assembly - Common.
2. Extend IEnumerable
3. Introduce overload with index.